docs/downloads.md

```diff
@@ -1,12 +1,37 @@
 # OpenMS Installers
+::::{tab-set}
 
+:::{tab-item} Release
+:sync: Release
 | Platform | Name | SHA256 Hash |
 |----------|------|-------------|
 | Windows   | {{ '[OpenMS-{0}-Win64.exe](https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/{0}/OpenMS-{0}-Win64.exe)'.format(version) }} | `d203985c7042b885ac1085c30a2d9f36d7609b47`|
 | macOS | [OpenMS-2.8.0-macOS.dmg](https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/2.8.0/OpenMS-2.8.0-macOS.dmg) | `d203985c7042b885ac1085c30a2d9f36d7609b47` |
 | GNU/Linux | [OpenMS-2.8.0-Debian-Linux-x86_64.deb](https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/2.8.0/OpenMS-2.8.0-Debian-Linux-x86_64.deb) | `d203985c7042b885ac1085c30a2d9f36d7609b47` |
 | Source | [OpenMS-2.8.0-src.tar.gz](https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/2.8.0/OpenMS-2.8.0-src.tar.gz) | `d203985c7042b885ac1085c30a2d9f36d7609b47` |
+:::
 
+:::{tab-item} RC
+:sync: RC
+| Platform | Name | SHA256 Hash |
+|----------|------|-------------|
+| Windows   | {{ '[OpenMS-2.8.0-Win64.exe](https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/RC/{0}/OpenMS-{0}-Win64.exe)'.format(version) }} | `d203985c7042b885ac1085c30a2d9f36d7609b47`|
```

Comment:
Please use variables for the version.

Reply:
The version variable is currently incorrect (2.7.0). I assume because our windows release for 2.8 is still broken. I can either bump the version to 2.8.0 and manually specify the windows release version, or we could wait until the 3.0.0 release

Reply:
Yes as long as all of the versions use variables, we can easily switch later. That's why I prefer variables